test/testdata/infer/generics/isa_with_type_member.rb

```diff
@@ -0,0 +1,99 @@
+# typed: true
```

Comment:
We don't have any tests for the special stdlib defaulting behavior. Do you think it's worth having? Or do you think it would fail other parts of the test suite?

Reply:
I'm not sure we need them, as there are existing tests that validate the behavior. Disabling the special case causes ~6 test failures that all relate to the uses of the stdlib generics.